src/catkin_pkg/package.py

```diff
@@ -166,6 +166,18 @@ def get_build_type(self):
             return build_type_exports[0]
         raise InvalidPackage('Only one <build_type> element is permitted.', self.filename)
 
+    def get_build_types(self):
```

Comment:
@cottsay should we rename this to something like `get_unconditional_build_types()` or somesuch to make it more obvious that this is an "advanced" method?

Should we also or alternatively prefix it with `_`?
```suggestion
    def _get_unconditional_build_types(self):
```

Reply:
I don't think it would hurt, but I also think that adding docs to the function would be sufficient.

Reply:
I did decide to rename it without the `_` prefix. Documentation for the method also indicates that conditions are not considered.